python/pyspark/sql/functions.py

```diff
@@ -1669,20 +1669,33 @@ def repeat(col, n):
     return Column(sc._jvm.functions.repeat(_to_java_column(col), n))
 
 
-@since(1.5)
+@since(2.4)
```

Comment:
I wouldn't change `since`. You can describe the behaviour changed by, for instance:

```python
        .. versionchanged:: 2.4
           The ``limit`` parameter blah blah..
```